src/Microsoft.Dnx.Runtime.Sources/Impl/EnvironmentNames.cs

```diff
@@ -16,12 +16,9 @@ internal static class EnvironmentNames
         public const string AppBase = "DNX_APPBASE";
         public const string Framework = "DNX_FRAMEWORK";
```

Comment:
This is separate (I called it out in the description) - we don't even pass it around too much - it didn't have to be env var at all.